src/Round.js

```diff
@@ -18,7 +18,9 @@ class Round {
   }
 
   startRound() {
-    let puzzleClass = new Puzzle(data);
+    let puzzleClass = new Puzzle(data.puzzles);
```

Comment:
`puzzleClass` is kind of a weird name for a variable. It is kind of going against that naming convention rule where we say **not** to include the data type of your value. I'd rename this to something like `currentPuzzle` or `selectedPuzzle`